addons/point_of_sale/static/src/app/debug/debug_widget.js

```diff
@@ -146,7 +147,7 @@ export class DebugWidget extends Component {
         }
     }
     get unpaidOrdersFilename() {
-        return `${this.env._t("unpaid orders")} ${moment().format("YYYY-MM-DD-HH-mm-ss")}.json`;
+        return `${this.env._t("unpaid orders")} ${DateTime.now().toFormat("yyyy-MM-dd-HH-mm-ss")}.json`;
```

Comment:
can't we use also serializeDatetime here? (same elsewhere)

Reply:
this is not the good format so no. 
const SERVER_DATE_FORMAT = "yyyy-MM-dd";
const SERVER_TIME_FORMAT = "HH:mm:ss";
const SERVER_DATETIME_FORMAT = `${SERVER_DATE_FORMAT} ${SERVER_TIME_FORMAT}`;

Reply:
Yes but we could serializeDateTime and replace spaces with dashes (i.e. through String.prototype.replace or a regex).

IMO it is better to have not too many occurrences of formatting strings in the codebase, just in case we would like to change it someday.

Reply:
Or we could ask the POS team if the dash between date and time really matters here